gio/src/io_extension.rs

```diff
@@ -0,0 +1,51 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+use glib::Type;
+use std::fmt;
+use std::ptr;
+
+/// The implementation of an `IOExtensionPoint`.
+#[doc(alias = "GIOExtension")]
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
```

Comment:
You might want to do a manual `Debug` impl that prints the name of the extension at least. That would seem useful for debugging purposes.

Reply:
Should I leave `Display` as is, or should I change that to show the name as well?

Reply:
The display seems fine to me

Reply:
Whatever you prefer, personally I don't think any of these types should implement `Display`...